content/en/serverless/distributed_tracing/serverless_trace_merging.md

```diff
@@ -0,0 +1,94 @@
+
+There are two primary reasons for instrumenting both `dd-trace` and AWS X-Ray tracing libraries:
+- In a serverless-first environment, your application requires AWS X-Ray active tracing in AWS managed services such as API Gateway or Step Functions, you are tracing your Lambda functions with `dd-trace`, and you want to visualize connected traces across your serverless resources.
+- In a hybrid environment with both Lambda functions and hosts, `dd-trace` is installed on your hosts with the Datadog Agent, your serverless functions are traced with AWS X-Ray, and you want to visualize connected traces for transactions across Lambda functions and hosts.
+
+Note that this may result in higher usage bills. X-Ray spans will continue to be available in your merged traces after 2-5 minutes. In many cases, we recommend only using a single tracing library. Learn more about choosing your tracing library [here][1].
+
+You can find setup instructions for each of the above use cases below:
+
+- [Trace merging in a serverless-first environment](#trace-merging-in-a-serverless-first-environment)
+- [Trace merging across AWS Lambda and hosts](#trace-merging-across-aws-lambda-and-hosts)
+
+### Trace merging in a serverless-first environment
+
+AWS X-Ray provides both a backend AWS service (AWS X-Ray active tracing) and a set of client libraries. [Enabling the backend AWS service alone in the Lambda console][2] gives you `Initialization` and `Invocation` spans for your AWS Lambda functions. You can also enable AWS X-Ray active tracing from the API Gateway and Step Function consoles.
+
+Both the AWS X-Ray SDK and Datadog APM client libraries (`dd-trace`) add medatada and spans for downstream calls by accessing the function directly. Assuming you are using `dd-trace` to trace at the handler level, your setup should be similar to the following:
+
+1. You have enabled [AWS X-Ray active tracing][2] on your Lambda functions from the AWS Lambda console and our [AWS X-Ray integration within Datadog][3].
+2. You have instrumented your Lambda functions with Datadog APM (`dd-trace`) by following the [installation instructions for your Lambda runtime][4].
+3. The AWS X-Ray client libraries should not be installed on your Lambda functions. Otherwise, you will see duplicate traces.
+
+To merge the traces, set the corresponding environment variable as outlined below:
+
+{{< tabs >}}
+{{% tab "Node.js" %}}
+```javascript
+module.exports.hello = datadog(
+    (event, context, callback) => {
+        longCalculation();
+
+        callback(null, {
+            statusCode: 200,
+            body: 'Hello from serverless!'
+        });
+    },
+    { mergeDatadogXrayTraces: true }
+);
+```
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+Set the `DD_MERGE_XRAY_TRACES` environment variable to `True` on your Lambda function.
```

Comment:
Not sure if this section deserves to be broken down by language. Perhaps we could say "Set the `DD_MERGE_XRAY_TRACES` (`DD_MERGE_DATADOG_XRAY_TRACES` for Ruby) environment variable to `true` on your Lambda function"?

Probably we could also mention the option from serverless plugin and macro if the customer is using those?